### ISSUE TITLE
Subfigures in appendix

### ISSUE THREAD

**Issue by da5nsy** [roaldarbol/LaPreprint#41]
I use the `subfigure` command in the main text, but I haven't been able to make it work in the appendix.

MWE: https://www.overleaf.com/3229474442wzpbfnjpvtwb
See lines 63-82 of `introduction.tex` for successful use (quoted below for future use), and lines 31-48 of `supplementary/appendices.tex` for unsuccessful attempt at using the same thing in the appendix.

- Am I doing something wrong? / Is there an easy workaround?
- If this is not possible given the current way that appendices are handled, are there alternative ways that I could handle appendices?

---

```
\begin{figure}
    \centering
        \begin{subfigure}[b]{0.3\textwidth}
            \centering
            \includegraphics[width=\linewidth]{src/figures/brighton.jpeg}
        \end{subfigure}
        \hfill
        \centering
            \begin{subfigure}[b]{0.3\textwidth}
            \centering
            \includegraphics[width=\linewidth]{src/figures/brighton.jpeg}
        \end{subfigure}
        \hfill
        \centering
            \begin{subfigure}[b]{0.3\textwidth}
            \centering
            \includegraphics[width=\linewidth]{src/figures/brighton.jpeg}
        \end{subfigure}
    \caption{\textbf{Lorem.} Ipsum!}
\end{figure}
```

**Comment by roaldarbol** [comment]
Yeah, they are a bit convoluted and messy at the moment. Restructuring them is on the map, but I haven't had the time yet (see https://github.com/roaldarbol/LaPreprint-class/issues/2 and https://github.com/roaldarbol/LaPreprint-class/issues/6 for reference).

Sorry, about that!

**Comment by da5nsy** [comment]
I'd potentially be able to offer some time/energy to this. Let me know if there are any sensible first steps that I could explore.

**Comment by hvoss49** [comment]
In an appendix the use of floating environments do not really make sense. You can use simple minipages:

```
\documentclass{article}
\usepackage[demo]{graphicx}
\usepackage{subcaption}

\begin{document}
\section{default main text body}

\begin{figure}[!htb]
	\begin{subfigure}[b]{0.3\textwidth}
		\includegraphics[width=\linewidth]{src/figures/brighton.jpeg}
	\end{subfigure}
	\hfill
	\begin{subfigure}[b]{0.3\textwidth}
		\includegraphics[width=\linewidth]{src/figures/brighton.jpeg}
	\end{subfigure}
	\hfill
	\begin{subfigure}[b]{0.3\textwidth}
		\includegraphics[width=\linewidth]{src/figures/brighton.jpeg}
	\end{subfigure}
	\caption{\textbf{Lorem.} Ipsum!}
\end{figure}
	
\appendix
\section{Appendix}
\noindent
\begin{minipage}{\textwidth}
\includegraphics[width=0.3\linewidth]{src/figures/brighton.jpeg}
\hfill
\includegraphics[width=0.3\linewidth]{src/figures/brighton.jpeg}
\hfill
\includegraphics[width=0.3\linewidth]{src/figures/brighton.jpeg}

\captionof{figure}{\textbf{Lorem.} Ipsum!}
\end{minipage}

\end{document}
```

<img width="373" alt="Bildschirmfoto 2023-06-27 um 18 29 32" src="https://github.com/roaldarbol/LaPreprint/assets/12891199/6fc6ec85-3eee-4dbd-8240-99987066babf">

**Comment by roaldarbol** [comment]
Hi both! Thanks a bunch! Feel free to play around with it some more. Do you have any preferences to how appendices look the best in a PDF? I like the current version, but the boxes can be a bit constraining, and I wonder whether it's worth it to keep them (we could avoid this altogether). I can also see that the newer eLife papers have stopped using them too.

**Comment by da5nsy** [comment]
I do think it's nice to have a visual difference for the appendices, but I don't have strong feelings about the boxes specifically

**Comment by da5nsy** [comment]
Any suggestions on how to get the automatic lettering for the subfigures in the appendix environment folks?

**Comment by da5nsy** [comment]
Just spotted another issue - when cross-referencing figures from the supplement the way they are referred to does not distinguish them from the main figures.

For example \autoref{fig:maintext1} and \autoref{fig:supp1} will both be rendered as "Figure 1", which is a bit of a disaster imo.

(Workaround suggestions welcome!)

**Comment by roaldarbol** [comment]
Sorry about the lack of responsiveness, my priorities have simply been elsewhere for a while. 

Not great that they share names, hard agree! I hope to find a bit of time over the next months to look into it.
